.github/workflows/codecov.yml

```
@@ -0,0 +1,24 @@
+# This is based on:
+# https://github.com/codecov/example-java-maven/blob/main/.github/workflows/ci.yml
+name: Codecov
+on: [push, pull_request]
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Set up JDK 17
+      uses: actions/setup-java@v1
+      with:
+        java-version: 17
+    - name: Install dependencies
+      run: mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
+    - name: Run tests and collect coverage
+      run: mvn -B test
```

```suggestion
      run: mvn -B prepare-package
```

This should be the `phase` name of the [maven execution phase](https://github.com/google/fhir-data-pipes/blob/62a2ffb824b432b0906c822bc60c74cea83193b1/coverage/pom.xml#L99).

---

Thanks for pointing this out, I actually changed the phase declaration for jacoco report to be `test` instead (I prefer to stick with `test` as it is faster).